developer-docs/latest/installation/cli.md

```diff
@@ -56,7 +56,7 @@ npx create-strapi-app my-project --quickstart
 ::::
 
 ::: tip
-If you want to use specific database, you don't have to use the `--quickstart` flag. The CLI will let you choose the database of your choice. The --quickstart flag sets the database  to SQLite.
+If you want to use specific database, you don't have to use the `--quickstart` flag. The CLI will let you choose the database of your choice. The `--quickstart` flag sets the database  to SQLite.
```

Comment:
Can you do me a small favor and delete the extra spaces at the end of this sentence where it says `database  to SQLite`

You didn't put them there, but it would be one less PR I need to make ;)